lib/typinggame_server/repositories/players_rooms_repository.rb

```diff
@@ -0,0 +1,9 @@
+class PlayersRoomsRepository < Hanami::Repository
+  def find_players_in_room(room_id:)
+    players_rooms.where(room_id: room_id).to_a
```

Comment:
This will return a `PlayerRoom` instead of a `Player`, which is _fine_, if that's what we want, but we might want to call this something like `by_room_id` to communicate that we're returning the model at hand.

Reply:
We likely want to have our query to fetch a player live on the `PlayerRepository` instead.

Reply:
@jgayfer discussed in slack and provided more clarity to what this is achieved here as you are correct, we are getting a `player_room` record rather than a `Player` or `Room` record.